src/schema.ts

```diff
@@ -155,7 +155,7 @@ export class Schema {
       const maxbins: any = bin.maxbins;
       if (!fieldSchema.binStats[maxbins]) {
         // need to calculate
-        fieldSchema.binStats[maxbins] = binSummary(maxbins, fieldSchema.stats);
+        fieldSchema.binStats[maxbins] = binSummary(maxbins, fieldSchema.stats, false);
```

Comment:
Looks like all occurrences of `binSummary` call this with `excludeInvalid=false` ignoring the input `excludeInvalid`.    I don't understand why it has to always be false.  Plus, if you don't wanna use it, why adding this parameter to `binSummary` method at all?